src/ui/server/content-strings/error-messages.ts

```diff
@@ -209,6 +209,14 @@ export const ERROR_MESSAGES = {
           },
         },
       },
+      SIGN_IN: {
+        [FIELD_IDS.INSURANCE.ACCOUNT.EMAIL]: {
+          INCORRECT: "Email address - either the email address or password you've entered is incorrect",
+        },
+        [FIELD_IDS.INSURANCE.ACCOUNT.PASSWORD]: {
+          INCORRECT: "Password - either the email address or password you've entered is incorrect",
```

Comment:
is there a reason for the "Password - " bit?

Reply:
Design. I presume it's because in the summary, without this, it doesn't have the context of what field as both fields have the same error message